gateway/review/service.go

```diff
@@ -158,12 +177,41 @@ func (s *Service) Review(ctx *storagev2.Context, reviewID string, status types.R
 		return nil, fmt.Errorf("saving review error: %v", err)
 	}
 
-	if rev.Status == types.ReviewStatusApproved || rev.Status == types.ReviewStatusRejected {
+	// Fetch the session information
+	session, err := pgsession.New().FetchOne(ctx, rev.Session)
+	if err != nil {
+		return nil, fmt.Errorf("fetch session error: %v", err)
+	}
+
+	issueInfos := jira.AddNewReviewByUserIssueTemplate{
+		ReviewerName:         rev.ReviewOwner.Name,
+		ReviewApprovedGroups: strings.Join(approvedGroups, ", "),
+		ReviewStatus:         string(rev.Status),
+	}
+
+	jira.AddReviewByUserJiraIssue(ctx.OrgID, session.JiraIssue, issueInfos)
```

Comment:
I recommend logging the error.
Add a _ to indicate that you won't be dealing with the error.